alignak_module_ws/etc/arbiter/modules/mod-ws.cfg

```diff
@@ -38,7 +38,11 @@ define module {
     # ---------------
     # Set this variable to 1 to allow the host creation by the WS
     #allow_host_creation     0
+    #ignore_unknown_host     0
```

Comment:
Perhaps not use negative variable because complex to read code `not ignore_unknown_host` is double negation.
I propose replace with `allow_unknown_host`

Reply:
I think it is better to be clear in the configuration ... I want to *ignore unknown host* so I set `ignore_unknown_host` 😉  ... it is easier for the one who configures the application.

Reply:
Ok